presto-tests/src/test/java/io/prestosql/tests/TestJoinQueriesWithoutDynamicFiltering.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.tests;
+
+import com.google.common.collect.ImmutableMap;
+import io.prestosql.testing.AbstractTestJoinQueries;
+import io.prestosql.testing.QueryRunner;
+import io.prestosql.tests.tpch.TpchQueryRunnerBuilder;
+import org.testng.annotations.Test;
+
+/**
+ * @see TestJoinQueries for tests with dynamic filtering enabled
+ */
+public class TestJoinQueriesWithoutDynamicFiltering
```

Comment:
I'm not sure we really care about this setup. Maybe add a `TODO: remove after few releases`

Reply:
DF is pretty invasive, affects certain code areas, like CBO, reorder joins, PPD.
As long as we have a toggle to turn it on/off, we should have tests for both scenarios.

Reply:
> As long as we have a toggle to turn it on/off, we should have tests for both scenarios.

We have plenty of feature toggles in `FeaturesConfig` yet we don't do large scale tests with them on/off.

Reply:
I can't name a feature toggle that impacts as many code areas as this one.
If there are such, we should do large scale tests for them too.